mainapp/models.py

```diff
@@ -199,7 +201,8 @@ class Contributor(models.Model):
         verbose_name="District - ജില്ല"
     )
     name = models.CharField(max_length=100, verbose_name="Name - പേര്")
-    phone = models.CharField(max_length=10, verbose_name="Phone - ഫോണ്‍ നമ്പര്‍", validators=[RegexValidator(regex='^[6-9]\d{9}$', message='Please Enter 10 digit mobile number', code='invalid_mobile')])
+    phone_number_regex = RegexValidator(regex='^((\+91|0)[\- ]{0,1})?[456789]\d{9}$', message='Please Enter 10 digit mobile number or landline as 0<std code><phone number>', code='invalid_mobile')
```

Comment:
I'm sure this line can also be written once and be used in every class. That is, phone_number_regex doesn't have to be a class attribute.

Reply:
See https://github.com/IEEEKeralaSection/rescuekerala/pull/657#discussion_r211095554 for reasoning.

Reply:
Cool